### ISSUE TITLE
There is a way to lose a window forever

### ISSUE THREAD

**Issue by tsujan** [WayfireWM/wayfire#1569]
**To Reproduce**
1. Go to the first workspace (desktop), while you have more that one.
2. Open a window that's not too big.
3. Throw it to the left while grabbing its title-bar, but release it before reaching the left side of the screen.

Now, if you've done 3 with enough speed (it isn't easy but quite possible), the window slides beyond the left side of the screen, and you have no way of finding it. 

If you do 3 on desktop n > 1, it'll go to desktop n-1, and you'll find it there. But sliding leftward on the first desktop can result in its disappearance: it won't appear on the last desktop.

**My enabled plugins**
alpha, animate, autostart, command, decoration, expo, fast-switcher, grid, idle, move, place, resize, switcher, vswitch, window-rules, wobbly, wrot, zoom, cube, blur, workarounds, vswipe, wm-actions, wf-info

**Wayfire version**
0.7.4 (latest release at this moment)

**Comment by ammen99** [comment]
I wonder whether this is because of wobbly, could you try with wobbly disabled?

**Comment by tsujan** [comment]
Yes, it's because of the wobbly effect. I can't reproduce it without it.

However — apart from my report — it's natural to expect that, if a part of a window lies beyond the left side of the first desktop, it should be visible on the last desktop (a kind of "wrapping around"), as is the case with desktop n>1. If I correctly recall, it was so with Compiz.

KWin doesn't do it with any desktop; so, there's no expectation.

**Comment by tsujan** [comment]
And, of course, the same is true with the right side of the last desktop.
